### PR TITLE
Change the links to absolute so they work on supermarket

### LINKED ISSUE #147: README.md links don't work on supermarket.chef.io
### Cookbook version
current

### Chef-client version
n/a

### Platform Details
n/a

### Scenario:
Read the links

### Steps to Reproduce:
Look at the cookbook description on supermarket.chef.io.
### Expected Result:
Should be able to follow the links and read the documentation.

### Actual Result:
Semi ugly text is presented.

### PR DESCRIPTION
**Pull request by MarkGibbons**
### Description

Changes the style of links to documentation in hopes it will fix the link display in supermarket.
Supermarket doesn't support relative link values.

### Issues Resolved

#147

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

---
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/line/148)

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by damacus — APPROVED** [review]

File: spec/cookbook/documentation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  md = %r{\]\(https://github.com/sous-chefs/line/blob/master/(.*)\)}.match(line)
+  md ? md[1] : nil
+end
+describe 'linked documentation files exist' do
+  line_files.compact.each do |file|
+    it "File #{file} should exist" do
+      expect(File.exist?(file)).to eq true
+    end
+  end
+end
```
Comment (damacus) [inline spec/cookbook/documentation_spec.rb:15]: This seems a little OTT, but 

giphy

**Review by damacus — APPROVED** [review]

File: spec/cookbook/documentation_spec.rb (hunk 1 above)
Comment (MarkGibbons) [reply spec/cookbook/documentation_spec.rb:15]: I sort of think so too. The first error we encountered with putting documentation into a separate directory was that the files weren't there.  Changing to use the links to the master branch means we'll miss accidental file deletes or moves.  So I'm checking.

Comment (damacus) [reply spec/cookbook/documentation_spec.rb:15]: I wonder if we cna move this test out of the cookbook, and into the circleci pipeline so we can all reuse it?

For now, this is just an ace example

Comment (MarkGibbons) [reply spec/cookbook/documentation_spec.rb:15]: Let's see if this actually fixes the problem in supermarket.  Cheers

**Pull request merged**